### PR TITLE
doc(v1.7.0): update "Important Notes"

### PR DESCRIPTION
**Pull request by derekbit**
#### Which issue(s) this PR fixes:

Longhorn/longhorn#7672
Longhorn/longhorn#7534
Longhorn/longhorn#7641
Longhorn/longhorn#9054
Longhorn/longhorn#7927
Longhorn/longhorn#7070
Longhorn/longhorn#2856
Longhorn/longhorn#6205
Longhorn/longhorn#2765

### CODE REVIEW

File: content/docs/1.7.0/deploy/important-notes/index.md [hunk 1]
```diff
@@ -104,161 +62,78 @@ For example, adding the following labels to the namespace that is running Longho
...
+Filesystem trim is supported since Longhorn v1.7.0. If a disk is managed by the SPDK AIO bdev driver, the Trim (UNMAP) operation is not recommended in a production environment (ref). It is recommended to manage a block-type disk with an NVMe bdev driver.
 
-### Updating the Linux Kernel on Longhorn Nodes
+### Linux Kernel on Longhorn Nodes
 
-Host machines with Linux kernel 5.15 may unexpectedly reboot when volume-related IO errors occur. Update the Linux kernel on Longhorn nodes to version 5.19 or later to prevent such issues. For more information, see [Prerequisites](../../v2-data-engine/prerequisites/).
+Host machines with Linux kernel 5.15 may unexpectedly reboot when volume-related IO errors occur. To prevent this, update the Linux kernel on Longhorn nodes to version 5.19 or later. For more information, see [Prerequisites](../../v2-data-engine/prerequisites/). Additionally, version 6.7 or later is highly recommended for improved system stability.
 
-### Storage Network Support for Read-Write-Many (RWX) Volumes
+### Snapshot Creation Times Change from Time to Time
```
Comment (jhkrug) [inline content/docs/1.7.0/deploy/important-notes/index.md:129]: I had difficulty understanding this and line 131. Hopefully, I have understood the original issue sufficiently to suggest rewording.

```suggestion
### Snapshot Creation Time as shown in the UI occasionally changes
```

File: content/docs/1.7.0/deploy/important-notes/index.md [hunk 2]
```diff
@@ -104,161 +62,78 @@ For example, adding the following labels to the namespace that is running Longho
...
 
-Starting with Longhorn v1.7.0, the [storage network](../../advanced-resources/deploy/storage-network/) supports RWX volumes. However, the network's reliance on Multus results in a significant restriction.
+Snapshots created before Longhorn v1.7.0 changes from Time to Time. This issue arises because the engine randomly selects a replica and its snapshot map each time the UI requests snapshot info, leading to potential 1-second gaps between snapshots among different replicas. This bug was fixed in v1.7.0, but snapshots created before this version may still encounter the issue. For more information, see this [link](https://github.com/longhorn/longhorn/issues/7641).
```
Comment (jhkrug) [inline content/docs/1.7.0/deploy/important-notes/index.md:131]:
```suggestion
Snapshots created before Longhorn v1.7.0 have their Creation Time, as shown in the UI, change occasionally. This issue arises because the engine randomly selects a replica and its snapshot map each time the UI requests snapshot info, leading to potential 1-second gaps between snapshots among different replicas. This bug was fixed in v1.7.0, but snapshots created before this version may still encounter the issue. For more information, see this [link](https://github.com/longhorn/longhorn/issues/7641).
```

File: content/docs/1.7.0/deploy/important-notes/index.md [hunk 3]
```diff
@@ -8,73 +8,31 @@ Please see [here](https://github.com/longhorn/longhorn/releases/tag/v{{< current
...
+- [Networking](#networking)
+  - [Storage Network Support for Read-Write-Many (RWX) Volumes](#storage-network-support-for-read-write-many-rwx-volumes)
+- [Resilience](#resilience)
+  - [RWX Volumes Fast Failover](#rwx-volumes-fast-failover)
+  - [Support Configurable Timeout for Replica Rebuilding and Snapshot cloning](#support-configurable-timeout-for-replica-rebuilding-and-snapshot-cloning)
+- [Data Integrity and Reliability](#data-integrity-and-reliability)
+  - [Support Periodic and On-Demand Full Backups to Enhance Backup Reliability](#support-periodic-and-on-demand-full-backups-to-enhance-backup-reliability)
+  - [High Availability of Backing Images](#high-availability-of-backing-images)
 - [V2 Data Engine](#v2-data-engine)
   - [Longhorn System Upgrade](#longhorn-system-upgrade)
```
Comment (DamiaSan) [inline content/docs/1.7.0/deploy/important-notes/index.md:22]: Better _Longhorn System Live Upgrade_?

File: content/docs/1.7.0/deploy/important-notes/index.md (hunk 2 above)
Comment (DamiaSan) [inline content/docs/1.7.0/deploy/important-notes/index.md:131]: Also when a volume rebuild is triggered the snapshot creation time changes

**Review by DamiaSan — COMMENTED** [review]
LGTM, just a couple of comments.

File: content/docs/1.7.0/deploy/important-notes/index.md [hunk 4]
```diff
@@ -104,161 +62,78 @@ For example, adding the following labels to the namespace that is running Longho
...
+Filesystem trim is supported since Longhorn v1.7.0. If a disk is managed by the SPDK AIO bdev driver, the Trim (UNMAP) operation is not recommended in a production environment (ref). It is recommended to manage a block-type disk with an NVMe bdev driver.
 
-### Updating the Linux Kernel on Longhorn Nodes
+### Linux Kernel on Longhorn Nodes
 
-Host machines with Linux kernel 5.15 may unexpectedly reboot when volume-related IO errors occur. Update the Linux kernel on Longhorn nodes to version 5.19 or later to prevent such issues. For more information, see [Prerequisites](../../v2-data-engine/prerequisites/).
+Host machines with Linux kernel 5.15 may unexpectedly reboot when volume-related IO errors occur. To prevent this, update the Linux kernel on Longhorn nodes to version 5.19 or later. For more information, see [Prerequisites](../../v2-data-engine/prerequisites/). Additionally, version 6.7 or later is highly recommended for improved system stability.
```
Comment (jhkrug) [inline content/docs/1.7.0/deploy/important-notes/index.md:127]:
```suggestion
Host machines with Linux kernel 5.15 may unexpectedly reboot when volume-related IO errors occur. To prevent this, update the Linux kernel on Longhorn nodes to version 5.19 or later. For more information, see [Prerequisites](../../v2-data-engine/prerequisites/). Version 6.7 or later is recommended for improved system stability.
```

File: content/docs/1.7.0/deploy/important-notes/index.md [hunk 5]
```diff
@@ -104,161 +62,78 @@ For example, adding the following labels to the namespace that is running Longho
...
 
-Multus networks operate within the Kubernetes network namespace, so Longhorn can mount NFS endpoints only within the CSI plugin pod container network namespace. Consequently, NFS mount connections to the Share Manager pod become unresponsive when the CSI plugin pod restarts. This occurs because the namespace in which the connection was established is no longer available.
+### Unable to Revert a Volume to a Snapshot Created before Longhorn v1.7.0
+
+Reverting a volume to a snapshot created before Longhorn v1.7.0 is not supported due to an incorrect UserCreated flag set on the snapshot. A workaround is to back up the existing snapshots before upgrading to Longhorn v1.7.0 and restore them if needed. The bug is fixed in v1.7.0, and more information can be found [here](https://github.com/longhorn/longhorn/issues/9054).
```
Comment (jhkrug) [inline content/docs/1.7.0/deploy/important-notes/index.md:135]: A minor suggestion 'A workaround' suggests there may be more than one. Using 'The workaround' sounds more specific.

```suggestion
Reverting a volume to a snapshot created before Longhorn v1.7.0 is not supported due to an incorrect UserCreated flag set on the snapshot. The workaround is to back up the existing snapshots before upgrading to Longhorn v1.7.0 and restore them if needed. The bug is fixed in v1.7.0, and more information can be found [here](https://github.com/longhorn/longhorn/issues/9054).
```

**Review by jhkrug — CHANGES_REQUESTED** [review]

File: content/docs/1.7.0/deploy/important-notes/index.md (hunk 3 above)
Comment (derekbit) [reply content/docs/1.7.0/deploy/important-notes/index.md:22]: No, it is not related to live upgrade. Before introducing the control plane of live upgrade, users need to detach volumes.

**Review by jhkrug — DISMISSED** [review]
lgtm. thanks

File: content/docs/1.7.0/deploy/important-notes/index.md (hunk 2 above)
Comment (derekbit) [reply content/docs/1.7.0/deploy/important-notes/index.md]: Updated. Thank you.

File: content/docs/1.7.0/deploy/important-notes/index.md [hunk 6]
```diff
@@ -104,161 +62,78 @@ For example, adding the following labels to the namespace that is running Longho
...
 
-### Storage Network Support for Read-Write-Many (RWX) Volumes
+### Snapshot Creation Time as shown in the UI occasionally changes
 
-Starting with Longhorn v1.7.0, the [storage network](../../advanced-resources/deploy/storage-network/) supports RWX volumes. However, the network's reliance on Multus results in a significant restriction.
+Snapshots created before Longhorn v1.7.0 may change occasionally. This issue arises because the engine randomly selects a replica and its snapshot map each time the UI requests snapshot information or when a replica is rebuilt with a random healthy replica. This can lead to potential 1-second gaps between snapshots among different replicas. Although this bug was fixed in v1.7.0, snapshots created before this version may still encounter the issue. For more information, see this [link](https://github.com/longhorn/longhorn/issues/7641).
```
Comment (DamiaSan) [inline content/docs/1.7.0/deploy/important-notes/index.md:131]: _This can lead to potential 1-second gaps between snapshots among different replicas_
When the snapshot is rebuilt this is not true.

Comment (derekbit) [reply content/docs/1.7.0/deploy/important-notes/index.md:131]: Maybe`potential time gaps`?

**Review by DamiaSan — APPROVED** [review]

File: content/docs/1.7.0/deploy/important-notes/index.md
Comment (ejweber) [inline content/docs/1.7.0/deploy/important-notes/index.md:1]: The minimum XFS file system size behavior is new in v1.7.0. I added the note in https://github.com/longhorn/website/pull/905. IMO it should be in the important notes for this version. Is the omission accidental, or do you feel it should not be included?

**Review by ejweber — CHANGES_REQUESTED** [review]

File: content/docs/1.7.0/deploy/important-notes/index.md
Comment (derekbit) [reply content/docs/1.7.0/deploy/important-notes/index.md:1]: Yes, we need it. Sorry for mistakenly deleteing it. I added it back.

Comment (ejweber) [reply content/docs/1.7.0/deploy/important-notes/index.md:1]: No worries!

**Review by ejweber — APPROVED** [review]

**Review by james-munson — APPROVED** [review]
LGTM also
